Driver_FET.lib

```diff
@@ -701,6 +701,68 @@ X LDEL 9 -500 -600 100 R 50 50 1 1 P
 ENDDRAW
 ENDDEF
 #
+# HIP4082_DIP
+#
+DEF HIP4082_DIP U 0 20 Y Y 1 F N
+F0 "U" 100 650 50 H V L CNN
+F1 "HIP4082_DIP" 100 550 50 H V L CNN
+F2 "Package_DIP:DIP-16_W7.62mm" 100 750 50 H I L CNN
+F3 "" 100 750 50 H I C CNN
+$FPLIST
+ DIP-*_W7.62mm*
```

Comment:
No dashes or underscores. Underscores are to replaced with an asterisk and dashes are removed.